libethashseal/Ethash.cpp

```diff
@@ -84,10 +84,13 @@ StringHashMap Ethash::jsInfo(BlockHeader const& _bi) const
 
 EVMSchedule const& Ethash::evmSchedule(EnvInfo const& _envInfo) const
 {
-	if (_envInfo.number() >= chainParams().u256Param("frontierCompatibilityModeLimit"))
-		return HomesteadSchedule;
+	if (_envInfo.number() >= chainParams().u256Param("EIP150forkBlock"))
+		return DevconSchedule;
```

Comment:
This is an issue with the new GitHub review interface. On this page it shows old comments even the code has been already changed. It is better to see all changes in https://github.com/ethereum/cpp-ethereum/pull/3345/files.